src/app/core/authentication/component/auth/auth.component.scss

```diff
@@ -0,0 +1,184 @@
+@import '../../../../../styles.scss';
+
+*{
+    padding: 0;
+    box-sizing: border-box;
+    font-family: $font-primary, $font-secondary;
+}
+
+body{
+    width: 100vw;
+    height: 100vh;
+    background-image: url("/assets/blob-scene-haikei\ \(23\).svg");
+    background-size: cover;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+}
+
+.container{
+    background-color: $color-light;
+    border-radius: 30px;
+    box-shadow: 0 5px 15px rgba(0, 0, 0, 0.35);
+    position: relative;
+    margin: auto;
+    overflow: hidden;
+    width: 1226px;
+    height: auto;
+    max-width: 100%;
+    min-height: 480px;
+    h1{
+        color: $color-primary;
+    }
+    p, h1, h2{
```

Comment:
if the styles related to p, h1, h2 will be repeated across multiple areas of your project, we can consider adding a typography partial file in the styles directory (_typography.scss) and import it into our global styles